### DISCUSSION TITLE
Creating Redirection custom views

### DISCUSSION THREAD

**Discussion by primetoxinz** [requarks/wiki#3912] [Help / Questions]
Currently running 2.5.170 and went to make a Redirection custom view
![image](https://user-images.githubusercontent.com/4721642/117053909-87298480-ace7-11eb-920e-99752e3dffd3.png)
And found that the button does not work.

I went and investigated the [source](https://github.com/Requarks/wiki/blob/f55caab359ca7715012e3f9a3b9e31f16cccf399/client/components/editor/editor-modal-editorselect.vue#L126) and found that this action is commented out.

Is there a plan to readd this feature?

**Comment by gingerbeardman** [comment]
Great to read in https://github.com/Requarks/wiki/issues/4147 that this is slated for v3.0
